utils/zero.go

```diff
@@ -1,6 +1,8 @@
 package utils
 
-import "reflect"
+import (
+	"reflect"
```

Comment:
Why?

Please run `goimports` on this file

Reply:
I did. Neither does `goimports` complain nor does it update it. Either forms are valid.

Reply:
Ah. I misread this diff and falsely assumed that this PR adds the import (as opposed to reformatting).

This has been correct from the beginning.